ganttproject/src/net/sourceforge/ganttproject/export/ExporterToCSV.java

```diff
@@ -37,8 +41,52 @@
 import biz.ganttproject.impex.csv.GanttCSVExport;
 
 public class ExporterToCSV extends ExporterBase {
-  /** List of available/associated extensions */
-  private static String[] FILE_EXTENSIONS = new String[] { "csv" };
+  static class FileTypeOption extends GPAbstractOption<String> implements EnumerationOption {
+    static final String[] FILE_FORMAT_ID = new String[] { "impex.csv.fileformat.csv", "impex.xls.fileformat.xls" };
```

Comment:
I suggest replacing the first `csv` and `xls` with `spreadsheet`. This will allow i18n keys to be grouped together in a sorted order.